src/footer/styleMap.ts

```diff
@@ -0,0 +1,7 @@
+export const styleMap = {
+  footer: {
```

Comment:
suggestion: you can put this in our style.js file.  Most styles are there. I do believe we could break this up though, but in a style folder with separate files for each component's styles and then export them all from an index...

Reply:
Let's tackle this when with the style/UI refactoring work.